### ISSUE TITLE
Redundant / added in router while using strict routing

### ISSUE THREAD

**Issue by Alaskyh** [dimdenGD/ultimate-express#40]
I tried using SvelteKit's handler with `ultimate-express`, while serving the website on a different base path (`paths: { base: '/ui' }`).

Initially, it was presenting an infinite redirect because the `/` was being stripped from the path.
`app.set('strict routing', true)` solved that, and all good there.

However, this introduced another issue: After enabling it, it was 404'ing. After further inspection, seems like an extra slash is being added to the URL when it should not be there.

https://github.com/dimdenGD/ultimate-express/blob/b34c5b15ba16e5ad1963d4c45d5528360014f220/src/router.js#L454-L456

This whole piece of code feels redundant since the `/` is not stripped at all anywhere, not sure why you'd want the URL to become `//`. Commenting this out fixes my issue.

Keep in mind it works in express without using `app.set('strict routing', true)` at all.

TL;DR:
- `express`: `GET /ui/` -> `req.pathname` is `/ui/`
- `ultimate-express`, without `strict routing`: `GET /ui/` -> `req.pathname` is `/ui`
- `ultimate-express`, with `strict routing`: `GET /ui/` -> `req.pathname` is `/ui//`

**Comment by dimdenGD** [comment]
I couldn't reproduce your issue, but I noticed that `req.path` was implemented wrong and fixed it. Please try and check if it fixed it for you: `npm install https://pkg.pr.new/dimdenGD/ultimate-express@61150dd`

Also I assume you meant `req.path`? You have `req.pathname` in your message which doesn't exist in Express.

If it wasn't fixed, please provide a simple reproduction code. Also as for infinite redirect, if it still happens with new version, please open a second issue and provide reproduction code for it too.

**Comment by dimdenGD** [comment]
Fix now released on NPM (1.3.5)
